OpenSim/Examples/CustomActuatorExample/ControllableSpring.h

```diff
@@ -125,13 +125,13 @@ void computeForce(const SimTK::State& s,
     {
         pointA_inGround = _pointA;
         pointB_inGround = _pointB;
-        engine.transformPosition(s, engine.getGroundBody(), _pointA, *_bodyA, _pointA);
-        engine.transformPosition(s, engine.getGroundBody(), _pointB, *_bodyB, _pointB);
+        engine.transformPosition(s, getModel().getGround(), _pointA, *_bodyA, _pointA);
+        engine.transformPosition(s, getModel().getGround(), _pointB, *_bodyB, _pointB);
```

Comment:
these are prime candidates to use the new Frame::findLocationInAnotherFrame() and similar methods.